tests/Rule/EmailTest.php

```diff
@@ -53,7 +53,9 @@ public function getValidAddresses()
     {
         return [
             ['berry@github.com'],
-            ['berry+plus-sign@github.com.museum']
+            ['berry+plus-sign@github.com.museum'],
+            ['berry@githüb.com'],
+            ['bërry@github.com']
         ];
```

Comment:
You might want to add some more emailadresses to check, as well as some (much more) invalid e-mail adresses. Since this is quite a big change, you know :)

Reply:
Like @RobQuistNL says, this validation approach is the big change because the original approach use the native `filter` function to validate e-mail address. And the new change use the regular expression.

I agree with @RobQuistNL, and It should add more tests to prove the regular expression can validate e-mail address currently :+1:.